worker/earn.js

```diff
@@ -63,8 +63,8 @@ export async function earn ({ name }) {
           'day') uv
         JOIN users ON users.id = uv.id
         WHERE NOT (users.id = ANY (${SN_NO_REWARDS_IDS}))
+        AND uv.proportion >= 0.0000125
```

Comment:
Why 0.0000125 here? This look related to the number above but I am not sure proportion and minimum payout is related. Or similarity is just a coincidence?

Reply:
Rewards, `r`, are on the order of 100,000,000 msats. We don't want to reward people less than 1 sat, so what the smallest proportion, `p`, that when we account for referrals gives a reward of 1 sat?

`rp - .2rp = .8rp = 1000` then `p = 1000/.8r` then if we set `r = 100,000,000`, `p = 0.000025`